Tests/MeiliSearchUnitTests/SettingsTests.swift

```diff
@@ -17,7 +17,6 @@ class SettingsTests: XCTestCase {
             "exactness",
             "desc(release_date)"
         ],
-        "distinctAttribute": null,
```

Comment:
Why is this removed ?

Reply:
I removed since there's no usage of it whatsoever, if you think it is still relevant, I'll revert this change.

Reply:
You are right, it is missing.